pkg/gitreceive/build_test.go

```diff
@@ -0,0 +1,93 @@
+package gitreceive
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/arschles/assert"
+	"github.com/deis/builder/pkg"
+	"github.com/deis/builder/pkg/gitreceive/storage"
+)
+
+const (
+	bucketName = "mybucket"
+	objKey     = "myobj"
+)
+
+func TestGetProcFileFromRepoSuccess(t *testing.T) {
+	tmpDir, err := ioutil.TempDir("", "tmpdir")
+	if err != nil {
+		t.Fatalf("error creating temp directory (%s)", err)
+	}
```

Comment:
nitpick, but not blocking for this PR - you can shorten this next time to `assert.NoErr(t, err)`